app/task_routes.py

```diff
@@ -0,0 +1,114 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests
+import os
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message":f"{cls.__name__} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response({"message":f"{cls.__name__} {model_id} not found"}, 404))
+
+    return model
+
+def validate_info(request_obj):
+    obj_keys = request_obj.keys()
+    if 'title' not in obj_keys or 'description' not in obj_keys:
+        abort(make_response({"details": "Invalid data"}, 400))
+    else:
+        return request_obj
+
+
+def wrap_response(cls, response_obj):
+    return {f"{cls.__name__.lower()}": response_obj}
```

Comment:
Let's move these to a separate file for helper functions. That way the`routes.py` files will only have functions dedicated to routes.